.github/workflows/ci-workflow.yml

```diff
@@ -200,3 +200,20 @@ jobs:
 
       - name: Run zemu tests
         run: cd test && yarn test -t "\[Nano SP\]"
+
+  job_scan_build:
```

Comment:
So this is NanoS only scan ?

Reply:
It generally is. In this case, the only non-NanoS code is 
```
#ifdef TARGET_NANOX
                // grab the current plane mode setting
                G_io_app.plane_mode = os_setting_get(OS_SETTING_PLANEMODE, NULL, 0);
#endif  // TARGET_NANOX
```